sdk/cosmos/azure-cosmos-examples/src/test/java/com/azure/cosmos/rx/examples/CollectionCRUDAsyncAPITest.java

```diff
@@ -380,7 +381,7 @@ private DocumentCollection getMultiPartitionCollectionDefinition() {
         List<Index> indexes = new ArrayList<>();
         indexes.add(Index.range(DataType.STRING, -1));
         indexes.add(Index.range(DataType.NUMBER, -1));
-        includedPath.setIndexes(indexes);
+        ModelBridgeInternal.setIncludedPathIndexes(includedPath, indexes);
```

Comment:
We should change this. Since this is in examples module, we should not use `*BridgeInternal` classes here. 

Also please check if you have not un-intentionally made this change in other public surface area (examples sub-module)

Reply:
hmm, I need to change it because I hide the setIndexes method of IncludedPath, should I keep it public?

Reply:
chatted offline, make sense that do not call this new method here, since it is not accessible publicly. Will remove.

Reply:
Removed from all tests.